clients/da-vinci-client/src/main/java/com/linkedin/davinci/client/AvroGenericDaVinciClient.java

```diff
@@ -369,16 +369,43 @@ public CompletableFuture<Map<K, V>> batchGet(Set<K> keys) {
   }
```

Comment:
Comment in a random place to create a thread:
> Server-side or client-side?

This one adds server-side compute by invoking end server endpoints similar to routers. The client-side one will come later